src/ARMJIT_Memory.cpp

```diff
@@ -645,7 +645,7 @@ void Init()
     #ifdef __APPLE__
         char* fastmem_pid_name = new char[snprintf(NULL, 0, "%i", getpid())];
         sprintf(fastmem_pid_name, "%d", getpid());
-        strcat(fastmem_pid_name, "melondsfastmem);
+        strcat(fastmem_pid_name, "melondsfastmem");
```

Comment:
the strcat is in both cases unecessary. You just need to change the formatting string to "melondsfastmem%d". Also for consistency change it in both cases to %d

Reply:
Done.